ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -84,7 +84,10 @@
         {{ getRegistrationType(item.registrationType) }}
         <span v-if="isPpr && !isChild"> - Base Registration</span>
       </div>
-      <div v-else class="pr-2">{{ item.registrationDescription }}
+      <div v-else class="pr-2">{{ multipleWordsToTitleCase(item.registrationDescription, true) }}
```

Comment:
Why is there a need to capitalize the sentence? Was it a requirement? The ticket didn't say much.

Reply:
It matches all of the designs and I confirmed with Tracey that they intended it to be in title case.

Reply:
I this it should be added in content, instead of making title cases.

Reply:
What do you mean by added in content?

Reply:
The text should come from content/resources as it without manipulating title cases.

Reply:
Resolving to move ahead.